docs/resources/volume.md

```diff
@@ -19,7 +19,8 @@ A primary volume will be created with the server. If there is a need for additio
 data "ionoscloud_image" "example" {
     type                  = "HDD"
     cloud_init            = "V1"
-    location              = "us/las"
+    image_alias           = "ubuntu:latest"
+    location              = "us/ewr"
```

Comment:
same as above

Reply:
@cristiGuranIonos same as above, it doesn't matter when speaking about the image.

Reply:
@cristiGuranIonos done